CONTRIBUTING.md

```
@@ -37,56 +37,71 @@ This project uses a typical GitHub fork and pull request [workflow](https://guid
-* **Review Approved** - The development work has been reviewed and the code is ready to be merged.
-* **Complete** - The development work has been merged and the issue is resolved.
+## Code Review
 
-Note: A pull request must be submitted for all issue goals before an issue will be moved to the "under review" status. If any goals or acceptance criteria have not been met, then the user story will be commented on to provide feedback, and the issue will be returned to the "In progress" state.
+Contributors are encouraged to [review PRs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
 
-## Licenses and attribution
+The following are some guidelines to consider when performing code reviews.
 
-For complete attribution and licensing information for parts of the project that are not in the public domain, see the [LICENSE](LICENSE.md).
+- Tone
```

Markdownlint (0.35.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)